src/routes/upload.ts

```diff
@@ -1,36 +1,65 @@
 import { UPLOAD_PATH } from '../config';
 import { Router } from 'express';
-import { createWriteStream, existsSync, rmSync } from 'fs';
+import { createWriteStream, existsSync, mkdirSync, rmSync, statSync } from 'fs';
+const { execSync } = require('child_process');
 
 const router = Router();
+let fileSize: string | undefined;
 
 router.post('/', (req, res) => {
   try {
     req.pipe(req.busboy); // Pipe it trough busboy
 
+    req.busboy.on('field', (fieldname, file, data) => {
+      console.log(`Size of upload file is ${file}`);
+      fileSize = file;
+    });
     req.busboy.on('file', (fieldname, file, data) => {
       const filename = data && data.filename ? data.filename : 'upload.raucb';
       console.log(`Upload of '${filename}' started`);
       // Create a write stream of the new file
       try {
         try {
           if (existsSync(UPLOAD_PATH + filename)) {
```

Comment:
`UPLOAD_PATH + filename` repeated about 5 times,
so it will make sense for me to do:
`const uploadFilePath = UPLOAD_PATH + filename`

Reply:
updated